fontforge/python.c

```diff
@@ -4999,7 +5007,15 @@ return( NULL );
 	}
     }
     if ( c->closed ) {
+
 	SplineMake(ss->last,ss->first,c->is_quadratic);
+	if ( c->is_quadratic && (ss->last->nextcpselected || ss->first->prevcpselected) ) {
+            // The convention for tracking selection of quadratic control
+	    // points is to use nextcpselected except at the tail of the
+	    // list, where it's prevcpselected on the first point.
+	     ss->last->nextcpselected = false;
+	     ss->first->prevcpselected = true;
+	}
```

Comment:
The other part of the convention is handled by `RefigureSpline2()`. This code is needed because the last off-curve point is handled differently.